### ISSUE TITLE
Request: add arguments to launchbuddy to allow autolaunching a profile on startup

### ISSUE THREAD

**Issue by Snaid1** [TheCheatsrichter/Gw2_Launchbuddy#239]
I love Launchbuddy but when I want to play Guild Wars 2 just on one account instead of multiboxing I like to use a single shortcut since it launches faster and I don't have to do several extra clicks in order for it to launch. I've recently learned about several features in launchbuddy that I would love to take advantage of but still want to be able to use a single shortcut to launch the account. could an arguement or command line option be added so I could create a launchbuddy shortcut to autolaunch a profile? If this already exists could you explain how to use it/point me to the documentation? Thanks!

TLDR: I'd like to be able to make a shortcut to automatically launch a profile in launchbuddy just by clicking the shortcut.
